fastlane/Fastfile

```diff
@@ -270,7 +270,7 @@ end
 desc "Runs tests for v3 in Debug config"
 lane :test_v3 do |options|
 
-  setCIEnvironmentVariable("../Tests_v3/StreamChatTestPlan.xctestplan")
+  setCIEnvironmentVariable("../Tests_v3/StreamChatTests/StreamChatTestPlan.xctestplan")
```

Comment:
nit: I vote for `../Tests_v3/StreamChat/` other than `../Tests_v3/StreamChatTests/` as we are already in **Tests_v3** folder

Reply:
I named the folders after the **targets**, the files under `StreamChatTests` are files _only_ for that target. So files like `AssertAsync` will stay under `Tests_v3` since they'll (presumably) be used in both test targets.
Does this change your opinion about that? :D